src/Tests/dotnet-add-reference.Tests/GivenDotnetAddReference.cs

```diff
@@ -25,7 +25,7 @@ public class GivenDotnetAddReference : SdkTest
   Add a project-to-project reference to the project.
 
 Usage:
-  dotnet [options] add [<PROJECT>] reference <PROJECT_PATH>...
+  dotnet add [<PROJECT>] reference <PROJECT_PATH>... [options]
```

Comment:
I think we had an issue tracking this that we'll want to close when this goes in.

Reply:
I think I closed that issue when I checked in the main PR, System.CommandLine fixed it right before I finished that main PR.